packages/component-library/components/Button/styles.scss

```diff
@@ -1,8 +1,9 @@
-@import "../../styles/color";
-@import "../../styles/type";
-@import "../../styles/layout";
+@import "~@kaizen/component-library/styles/grid";
```

Comment:
@mbylstra

Reply:
Sorry, not sure what I'm meant to be looking at here.

Reply:
I reverted it to master and have only changed the mixin. Instead of adding a new mixin I've just grabbed the logic from `ca-type-ideal-button` and swapped out the mixin that was called.

Reply:
Ohh, Github pointed me the commit diff rather than the PR diff, which made it more confusing.

I didn't realise `kz-typography-inter` had the baseline shift, but I'm seeing it in Storybook, so 👍

Reply:
Oh, it's late in the day. This was the source of my confusion. I'd assumed that `kz-typography-inter` wouldn't have the baseline shift (seeing as it's kind of the way forward?), but it does by way of inheriting from `ca-type` which does all the baseline shift stuff.

Reply:
Oh, and the reason we left the baseline shift in `kz-typography-greycliff` because that was part of uplift work, and we thought removing the baseline shift would be too much effort to deal with in Big M.

We _could_ remove the baseline shift from `kz-typography-inter` seeing as it's not used in Big M, but I guess that would be confusing if `kz-typography-greycliff` still has it, and in any case could be left for another time. I'm thinking of when new components are built, but the heading or paragraph components aren't appropriate, so you use the mixins, but then the baseline shift would be unexpected.